src/pages/Home.jsx

```diff
@@ -28,25 +29,49 @@ const Home = () => {
   const isReallySmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
   const iconSize = isReallySmallScreen ? 'medium' : 'large';
 
+  // Logo section elements
+  const heading = 'PASS';
+  const punctuation = (
+    <Box component="span" sx={visuallyHidden}>
+      :
+    </Box>
+  );
+  const subheading = 'Personal Access System for Services';
+
   const logoSection = isReallySmallScreen ? (
-    <Stack justifyContent="center" alignItems="center" spacing={2} mb={2}>
-      <Typography variant="h1" fontWeight="500" fontSize="72px" color="primary">
-        PASS
-      </Typography>
-      <Box component="img" src="/assets/PASSLogolightmode.png" alt="PASS logo" width="50%" />
-    </Stack>
+    <Typography component="h1" color="primary">
+      <Stack component="span" justifyContent="center" alignItems="center" spacing={2} mb={2}>
+        <Typography variant="h1" component="span" fontWeight="500" fontSize="72px">
+          {heading}
+          {punctuation}
```

Comment:
I'm not quite sure I understand what this punctuation is doing. Could you describe it a bit?

Reply:
The punctuation is largely just a nice thing for screen reader users, so when they get to the `h1`, they don't hear "PASS Personal Access..." as all one sentence. The colon will usually provide a pause between "PASS" and "Personal...". Otherwise, assistive technologies may say "colon" literally, which is also descriptive.

Reply:
Is there a way to achieve this effect without writing screen reader specific code? Maybe wrap the section in a `header` tag and wrap the subtitle in a `p` element? https://mikebifulco.com/posts/semantic-html-heading-subtitle#use-this-code-to-add-subtitles-to-headings

PASS is volunteer-driven, and it doesn't have the resources to test PRs in multiple display environments. It's already a challenge to fully test our app in both desktop and mobile setups, so we can't guarantee that screen reader code will ever be validated, or maintained.

The ideal would be code that is fully agnostic to the display environment, and functions comparably in all environments. That way, a visual validation in the most recent version of Chrome can make us reasonably confident of the app's behavior in JAWS and NVDA. It's also easier to write automated tests for that kind of code.

I'm 100% on board with making PASS more usable for screen readers. We just need to do it in a way that's maintainable for a project this size. It's kind of a fun design challenge :)

Reply:
I hear you on maintainability. This code may not be the best solution (likely), but as you said, more maintainable and accessible solutions may require some significant changes elsewhere, when dealing with issues in the future.  

As for the h1/p suggestion, the text to be rendered needs to be _within_ the h1 for it to be presented to the screen reader. If someone hears “PASS”, it doesn’t actually say what it is. 

Another way would be leaving “PASS” out of the h1 and only containing the expanded acronym in the h1. Might be a bit cleaner overall.

Reply:
All this being said, I could leave some comments explaining the purposes of these elements.

Reply:
I cleaned it up a little and added a comment there. Happy to try something else, too, but this should be fairly simple to maintain, I think.

Reply:
@timbot1789

when you have a chance to re-look at this...

this PR is stalling #516

Reply:
This should be ok then. I think it would be good to go over accessibility standards with the project team at some point.

@milofultz  do you think you would be able to add [a small section to the testing wiki](https://github.com/codeforpdx/PASS/wiki/Testing) about testing the app with screenreaders and other assistive technologies? It's been a few years since I've done accessibility testing and I don't remember what the best practices are.